SPEC/FILES.md

```diff
@@ -759,6 +789,48 @@ A great source of [examples][] can be found in the tests for this API.
 
 The Mutable File System (MFS) is a virtual file system on top of IPFS that exposes a Unix like API over a virtual directory. It enables users to write and read from paths without having to worry about updating the graph. It enables things like [ipfs-blob-store](https://github.com/ipfs/ipfs-blob-store) to exist.
 
+#### `files.chmod`
+
+> Change mode for files and directories
+
+##### `ipfs.files.chmod(path, mode, [options])`
+
+Where:
+
+- `path` is the path to the entry to modify.  It might be:
+  - An existing MFS path to a file or a directory (e.g. `/my-dir/my-file.txt`)
+  - An IPFS path (e.g. `/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks`)
+  - A [CID][cid] instance (e.g. `new CID('QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks')`)
+- `mode` is the new file mode.  It might be:
+  - A string octal, e.g. `'0755'`
+  - A string modification of the existing mode, e.g. `'+x'`, `'-gw'`, etc
+  - An integer, e.g. the returned value from `parseInt('0755', 8)`
+- `options` is an optional Object that might contain the following keys:
+  - `recursive` is a Boolean value that indicates if `mode` should be applied to all sub files/directories of `path` (default: false)
+  - `format` is what type of nodes to write any modified entries as (default: `dag-pb`)
```

Comment:
Is this actually possible? What formats are supported? I’ve seen this in the docs for the other commands but always wondered...

Reply:
It's possible if you specify `dag-cbor` but the thing you'd get back probably isn't readable as you'd be expecting an instance of `DAGLink` from the `ipld-dag-pb` module and you'd get a plain JS object back.

Like this it's consistent with our other commands but it's a footgun so we should get rid of this everywhere and just make IPFS things be `dag-pb`.

Reply:
Can you remove from the docs here then and either also remove the others or make an issue so we can track it? 🙏

Reply:
I've pulled it out of this PR and opened #575 with removals from the other commands because `mode`/`mtime` discussions should not block that.